migrations/2_deploy_MyToken_1155.js

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config()
+var BigNumber = require('bignumber.js');
+
+const {TOKEN_BASE_URI, TOKEN_PRICE_BY_ETH, TOKEN_PRICE_BY_ERC20, COUNT_OF_SET, AMOUNT_LIMIT_OF_TOKEN} = process.env;
```

Comment:
You need to get ERC20 address from env and pass to the contract constructor in case if you are deploying to mainnet.